systemverilog-plugin/UhdmAst.cc

```diff
@@ -4384,6 +4384,9 @@ AST::AstNode *UhdmAst::process_object(vpiHandle obj_handle)
     case vpiInitial:
         process_initial();
         break;
+    case vpiFinal:
+        log_warning("Skipping unsupported object of type '%s'.\n", UHDM::VpiTypeName(obj_h).c_str());
```

Comment:
There is a function [`process_unsupported_stmt`](https://github.com/chipsalliance/yosys-f4pga-plugins/blob/b6a3874a3c8ff80fc3d999683db86fb7d8c1fc02/systemverilog-plugin/UhdmAst.cc#L4262-L4266)
Please modify it to add an optional argument `is_error = true` and something like:
```
const auto log_func = is_error ? log_error : log_warning;
log_func(/* arguments currently passed to log_error */);
```

Reply:
I applied this change but with a little tweak. Since `final_stmt` is a `vpiProcess`, it doesn't contain info about a filename and a line number. This results in a log like:
```
Warning: :0: Currently not supported object of type 'final_stmt'
```
I made a prefix dependent on `object->VpiLineNo()` since AFAIK it will always be non-zero if processed object contains data about filename and line number.
With this change it returns:
```
Warning: Currently not supported object of type 'final_stmt'
```
And in case there's a sufficient data:
```
ERROR: /hdd/yosys-systemverilog/Surelog/tests/ContAssign/dut.sv:5: Currently not supported object of type 'cont_assign'
```
(It's of course not a real error, we do support `cont_assign`, I forced it.)

Reply:
Awesome, thanks!